nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -75,21 +87,24 @@
             .defaultValue(MATCH_CRITERIA_ANY)
             .build();
     public static final PropertyDescriptor ATTRIBUTE_PATTERN = new PropertyDescriptor.Builder()
-            .name("Attribute Pattern")
+            .name("attribute-pattern")
+            .displayName("Attribute Pattern")
             .description("Regular Expression that specifies the names of attributes whose values will be matched against the terms in the dictionary")
             .required(true)
             .addValidator(StandardValidators.REGULAR_EXPRESSION_VALIDATOR)
             .defaultValue(".*")
             .build();
     public static final PropertyDescriptor DICTIONARY_FILE = new PropertyDescriptor.Builder()
-            .name("Dictionary File")
+            .name("dictionary-file")
+            .displayName("Dictionary File")
             .description("A new-line-delimited text file that includes the terms that should trigger a match. Empty lines are ignored.  The contents of "
                     + "the text file are loaded into memory when the processor is scheduled and reloaded when the contents are modified.")
             .required(true)
             .addValidator(StandardValidators.FILE_EXISTS_VALIDATOR)
             .build();
     public static final PropertyDescriptor DICTIONARY_FILTER = new PropertyDescriptor.Builder()
-            .name("Dictionary Filter Pattern")
```

Comment:
Must retain the previously existing name so as not to disturb existing configurations.  Andy's advice for the new property is good to follow going forward but it is wise to avoid changing old names.